libcxx/test/std/numerics/simd/simd.class/aliases.compile.pass.cpp

```diff
@@ -0,0 +1,47 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// <simd>
+
+// REQUIRES: std-at-least-c++26
```

Comment:
Recurring comment: let's please add a comment to describe what is being tested, like we do in the rest of the test suite.